src/java.desktop/share/classes/java/awt/color/ICC_Profile.java

```diff
@@ -107,30 +107,51 @@ public sealed class ICC_Profile implements Serializable
      */
     private transient volatile ProfileDeferralInfo deferralInfo;
 
+
+    /**
+     * Set to true for {@code BuiltInProfile}, false otherwise.
+     * This check is used in {@link #setData(int, byte[])} to prevent modifying
+     * BuiltInProfile.
+     */
+    private boolean isBuiltIn = false;
```

Comment:
Should the field be named `builtIn` instead? `isBuiltIn` starts with a verb which is used for naming methods.

Reply:
IIRC we usually have `is` or `has` prefixed to boolean vars as per naming convention?

Reply:
You are right, probably fields are named without the prefix and methods/local vars are prefixed with 'is'.